### ISSUE TITLE
Character seq2seq

### ISSUE THREAD

**Issue by Vanuan** [dmorr-google/wiki-reading#11]
Any hint on seq2seq implementation? I'm particularly interested in the character model as it doesn't require word tokenization. Could you summarise the layers required  please?

**Comment by danielhewlett** [comment]
Putting aside the pre-trained version, the character model is pretty much a standard seq2seq model:
1. The character embeddings are 30-dimensional vectors.
2. The encoder is a 2-layer RNN with GRU cells with a hidden state size of 1024, as noted in Section 4.1. 
3. The decoder is a single-layer RNN with the same type of cells.
4. The output at each step is computed by a softmax over the possible characters.
There is a diagram of the character-level seq2seq model in Figure 2, which indicates the parameter sharing between layers.
